### ISSUE TITLE
Update to Gaia DR3?

### ISSUE THREAD

**Issue by ethankruse** [afeinstein20/eleanor#256]
https://github.com/afeinstein20/eleanor/blob/cf19a998a99d0cea09c8426f8db2248fdc55f12c/eleanor/mast.py#L183

I don't know how big a deal this is, but should eleanor start using DR3? And update the docs to match? Or somehow try to support both DR2 and DR3?

I have no idea how many people actually even look at the Gaia header value or try to feed eleanor via Gaia IDs, but right now you have to give it DR2 IDs to get the correct source.

**Comment by afeinstein20** [comment]
good idea, Ethan! I'm going to incorporate this into our next version release.

**Comment by afeinstein20** [comment]
This has been implemented, so I am closing this issue. Thanks!
